L/libfreenect/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libfreenect"
+version = v"0.6.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/OpenKinect/libfreenect.git", "edad69caabe662e7996db9425c8b5c568efa275d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd libfreenect/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_CPP=OFF -DBUILD_EXAMPLES=OFF -DBUILD_FAKENECT=OFF -DBUILD_REDIST_PACKAGE=OFF ..
+make
+make install
+exit
```

Comment:
We need to add the [license file(s)](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Installing-the-license-file)
```suggestion
install_license APACHE20 GPL2
```